docs/addPythonLibs/index.md

```diff
@@ -0,0 +1,70 @@
+# The Python Environment for Questions
+
+This page is meant for discussion of details related to the Python environment that questions are run in, along with how to work within this environment for question development. Are you creating questions and finding that PrairieLearn does not have the Python libraries you need? Don't panic! There are a few different options you have for getting the necessary libraries installed that will be discussed below.
+
+## General Information
+
+For some general background, all `server.py` files for questions are run within a Docker container identical to the one used for local development. This container is using the Python version that is bundled with the [latest version of Miniconda](https://docs.conda.io/en/latest/miniconda.html) with packages in [this file](https://github.com/PrairieLearn/PrairieLearn/blob/master/images/plbase/python-requirements.txt) installed.
+
+### FAQ
+
+1. To see which packages are installed, simply run `pip list` in the docker container.
+
+2. To run a command line version of this Python environment, you may start it with the following command:
+
+   ```
+   docker run -it --rm prairielearn/prairielearn /bin/bash
+   ```
+
+3. To access files on the disk or in `serverFilesCourse` within `server.py`, get the directory from `data["options"]["question_path"]` and similar entries from the `data["options"]` dictionary.
+
+## Installing Libraries locally to your course
+
+The quickest way to add custom libraries is to install them directly to your course. This will assume that you have (at least passing) knowledge of using Git and Docker. If you are not familiar, it is recommended to follow the [Installing PL for local development](../installing.md) guide.
```

Comment:
```suggestion
The quickest way to add custom libraries is to install them directly to your course. This will assume that you are comfortable using Git and Docker. If you are not familiar, it is recommended to follow the [Installing PL for local development](../installing.md) guide.
```